app/models/assignment.rb

```diff
@@ -118,7 +118,7 @@ def self.evaluate_manually!(teacher_evaluation)
   end
 
   def content=(content)
-    if content.present?
+    unless exercise.reading?
       self.solution = exercise.single_choice? ? exercise.choice_index_for(content) : content
```

Comment:
That is a huge change! Can't this have subtle unexpected effects? Is this code  fixing an issue?

Reply:
it's not really so much fixing an issue as making one more transparent, and also lessening its effects.
Right now, if a user has a passed assignment on an exercise and sends an empty solution, the tests are run and the results are saved, while the solution's content is not, leading to inconsistencies.

Reply:
We could make this PR more clear by adding a test prior to this change in which we show that behavior. What do you think @julian-berbel?

Reply:
sounds good!